governator-test-junit/src/main/java/com/netflix/governator/guice/test/AnnotationBasedTestInjectorManager.java

```diff
@@ -90,16 +105,31 @@ private void inspectModulesForTestClass(Class<?> testClass) {
                 modulesForTestClass.add(moduleClass.newInstance());
             } catch (InstantiationException | IllegalAccessException e) {
                 try {
-                    Constructor<?> constructor = moduleClass.getDeclaredConstructors()[0];
-                    constructor.setAccessible(true);
-                    modulesForTestClass.add((Module) constructor.newInstance());
+                    Constructor<?>[] declaredConstructors = moduleClass.getDeclaredConstructors();
+                    for(int i=0; i < declaredConstructors.length; i++) {
+                        if(declaredConstructors[i].getParameterCount() == 0) {
+                            Constructor<?> zeroArgConstructor = declaredConstructors[i];
+                            zeroArgConstructor.setAccessible(true);
+                            modulesForTestClass.add((Module) zeroArgConstructor.newInstance());
+                            break;
+                        }
+                    }        
```

Comment:
If there's not a no-arg constructor this class will be silently skipped.  We should log an error or propagate the exception from getDeclaredConstructor() in my comment above.